### ISSUE TITLE
Could you please add BitMEX?

### ISSUE THREAD

**Issue by juanfran-s** [krijnsent/crypto_vba#56]
Hello,
Thank you very much for this project. It's really good.
Would you consider adding the BitMEX exchange?

https://www.bitmex.com/app/apiOverview

Best regards,
Juan

**Comment by krijnsent** [comment]
Hi Juan,
will have a look at it. They do have some code themselves, so it shouldn't be too hard for me to include it (https://github.com/BitMEX/api-connectors/tree/master/official-http/vba )
Cheers,
Koen

**Comment by krijnsent** [comment]
Hi there,
could you give my new code a try? It should basically work (tried it on TestNet).
Cheers,
Koen

**Comment by juanfran-s** [comment]
Hi Koen,
Thank you very much for adding it. I have tried to use it but I can't manage to make it work. I've tried with the TestBitmex() that comes in the module and also in another sheet using the endpoint walletSummary, that would give me the balance. Unfortunately, I always get the response:

{"error_nr":-2147483638,"error_txt":"The data necessary to complete this operation is not yet available. ","response_txt":0}"

Does it work for you? Any idea on what I can be doing wrong?

Best regards,
Juan
